cmake/install.cmake

```diff
@@ -87,19 +95,34 @@ ELSE()
   SET(INSTALL_PLUGINDIR_RPM "lib/mariadb/plugin")
 ENDIF()
 SET(INSTALL_INCLUDEDIR_RPM "include")
-SET(INSTALL_DOCDIR_RPM "docs")
+SET(INSTALL_DOCDIR_RPM "share/doc/mariadb-connector-odbc")
+SET(INSTALL_LICENSEDIR_RPM ${INSTALL_DOCDIR_RPM})
 SET(LIBMARIADB_STATIC_RPM "mariadbclient")
 
 #
 # DEB layout
-#
+# Only ia-32 and amd64 here. the list is too long to hardcode it
+IF(NOT CMAKE_LIBRARY_ARCHITECTURE) 
+  IF(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    SET(CMAKE_LIBRARY_ARCHITECTURE "x86_64-linux-gnu")
+  ELSE()
+    SET(CMAKE_LIBRARY_ARCHITECTURE "i386-linux-gnu")
+  ENDIF()
+ENDIF()
```

Comment:
is it needed? It's only used for DEB packages, do you mean that sometimes CMAKE_LIBRARY_ARCHITECTURE is unset on DEB?

Reply:
No, Otto said he uses Gentoo to build packages, so from my understanding install layout should depend only on passed DEB/RPM or remain default for bintar. but in fact debian variable contaning multiarch tuple seems  empty on all non-debians.
we can drop it because deb package will be built on Debians anyway

Reply:
No, I generally love this cmake feature that I can build any packages anywhere. I was building server rpms on my gentoo many times.

Okay, let's say that we only support building DEB packages on Gentoo for x86 and amd64.